src/main/java/com/glinka/touk_task/model/TicketType.java

```diff
@@ -0,0 +1,17 @@
+package com.glinka.touk_task.model;
+
+public enum TicketType {
+    ADULT(25),
+    STUDENT(18),
+    CHILD(12.5);
+
+    private double price;
```

Comment:
Widzisz jakieś zagrożenia związane z reprezentowaniem kwot w postaci `double`?
Jeżeli tak, to jakie i czego innego użył byś, żeby je zmitygować?